Documents/CommitteeSpecifications/2.1.0/sarif-schema-2.1.0.json

```diff
@@ -2702,7 +2702,7 @@
       "properties": {
 
         "guid": {
-          "description": "A stable, unique identifer for the suprression in the form of a GUID.",
+          "description": "A stable, unique identifer for the suppression in the form of a GUID.",
```

Comment:
Please create a different PR for changes on published artifacts (on docs.oasis-open.org i.e. standards work products.